test/integration/stats_integration_test.cc

```diff
@@ -252,6 +252,7 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithFakeSymbolTable) {
   // 2019/07/24  7503     43030       44000   add upstream filters to clusters
   // 2019/08/13  7877     42838       44000   skip EdfScheduler creation if all host weights equal
   // 2019/09/02  8118     42830       43000   Share symbol-tables in cluster/host stats.
+  // 2019/09/16  8100     42894       43000   Add transport socket matcher in cluster.
```

Comment:
i'm grabbing the number from the failed run of the ci, and trying to adjust these numbers as the comments indicates.

It's intuitively that added field in cluster.prot makes mem footprint larger. I'm not 100% sure this is the right number/changes to made though. 

[1] https://276470-65214191-gh.circle-artifacts.com/0/failed-testlogs/test/integration/stats_integration_test/shard_2_of_2/test.log
[2] https://276470-65214191-gh.circle-artifacts.com/0/failed-testlogs/test/integration/stats_integration_test/shard_1_of_2/test.log